src/lumigo_tracer/wrappers/http/sync_http_wrappers.py

```diff
@@ -122,9 +124,29 @@ def _update_request_data_increased_size_limit(http_info: dict, max_size: int) ->
     )
 
 
+def get_lumigo_connection_id(instance) -> Optional[int]:
+    return getattr(instance, LUMIGO_CONNECTION_ID_KEY, None)
+
+
+def set_lumigo_connection_id(instance):
+    r = random.random()
+    print("Choose random", r)
```

Comment:
Won't it print to the Cloudwatch of the customer?

Reply:
my bad sorry!!!!!